hondana/manga.py

```diff
@@ -234,18 +234,22 @@ def title(self) -> str:
         return title
 
     @property
-    def description(self) -> str:
+    def description(self) -> Optional[str]:
         """The manga's description/synopsis.
 
         Returns
         --------
-        :class:`str`
+        Optional[:class:`str`]
             The description of the manga, defaults to the ``en`` key in the titles.
-            Falls back to the next available key if ``en`` is not present.
+            Falls back to the next available key if ``en`` is not present. If there is
+            no description, returns None.
         """
         desc = self._description.get("en")
         if desc is None:
-            key = next(iter(self._description))
+            try:
+                key = next(iter(self._description))
+            except StopIteration:
+                return None
```

Comment:
```suggestion
            if not desc:
                return
```

I feel more explicitly checking the dict is Falsey is probably easier, since exception handling is not "free". A micro-optimization, I know. Let me know what you think.

Reply:
The reason I did exception handling is that most mangas won't have an empty description. It should be a performance optimization although this is Python and so I should probably use the cleaner syntax.

Reply:
Also this code is _very bad_ because it'll return None if `en` does not exist.

Reply:
This would be better:
```suggestion
            if not self._description:
                return None
            key = next(iter(self._description))
```